src/components/App/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
I guess your assignment was more heavy on the e2e testing with Cypress. So many thoughts on what tests are better to do. At our company we prioritize unit testing first. Youll find what work for you the best over your career.

Reply:
That literally got created with the boilerplate create-react-app and I missed removing this file. Won't happen again